src/components/Searcher.js

```diff
@@ -399,6 +399,15 @@ class Searcher extends React.Component<SearcherDefaultProps, SearcherProps, Sear
     };
   }
 
+  getRelevancyModels(): string[] {
```

Comment:
For the type "array of strings," we should be using the parameterized Flow type `Array<string>`...